Sources/TokamakCore/Fiber/FiberRenderer.swift

```diff
@@ -57,6 +57,11 @@ public protocol FiberRenderer {
     proposal: ProposedViewSize,
     in environment: EnvironmentValues
   ) -> CGSize
+
+  /// Run `action` on the next runloop.
+  ///
+  /// Called by the `FiberReconciler` to perform all reconciliation at once.
```

Comment:
could you elaborate in the comment what `at once` means in this context and why performing it all at once might be needed?